frontend/test/cypress/support/commands.ts

```diff
@@ -23,4 +23,22 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+/// <reference path="../support/index.d.ts" />
```

Comment:
What's this for?

Reply:
This was for the typescript compiler. Without it, the typechecker/linter was complaining. 

Got the solution from here: https://github.com/cypress-io/cypress-example-recipes/tree/master/examples/fundamentals__add-custom-command#adding-custom-commands